plancontext/service.go

```diff
@@ -10,54 +10,54 @@ import (
 )
 
 var (
-	serviceIDPath = cue.MakePath(
+	socketIDPath = cue.MakePath(
 		cue.Str("$dagger"),
 		cue.Str("service"),
```

Comment:
This can also be changed.

Reply:
Hmmm ... can it actually? That would break existing configurations

Reply:
Not on it's own. The cue definition also needs updating. I'll handle this.